distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated sources during queries
+
+Identity federation is the sharing of a user's identity and attributes among separate systems. In
+{branding}, it is used when performing federated queries - the user's identity is attached to outgoing
+requests, allowing federated sources to run queries as that user. This is critical because {branding}
+must make authorization decisions for all queries, not just those initiated by the local node.
+
+To represent a user's identity when federating, {branding} uses a SAML assertion. This is the format
+used internally to represent all subjects (even unauthenticated subjects). When federating over REST,
```

Comment:
I like how the word "subject" is capitalized later in the document. It calls attention to the fact that the word is used in a special way. Capitalize all such uses of "subject".

Reply:
Add the source word "endpoint" or "source" after "REST" because "REST" is more descriptive than it is substantive.

Also, I would capitalize the word "Source" to show that it represents a DDF concept and is not used generically.